sunpy/coordinates/frames.py

```diff
@@ -432,3 +439,122 @@ def calculate_distance(self):
         return self.realize_frame(SphericalRepresentation(lon=lon,
                                                           lat=lat,
                                                           distance=d))
+
+
+@add_common_docstring(**_variables_for_parse_time_docstring())
+class HeliocentricEarthEcliptic(SunPyBaseCoordinateFrame):
+    """
+    A coordinate or frame in the Heliocentric Earth Ecliptic system.
+
+    - The origin is the center of the Sun
+    - The x-axis is aligned with the Sun-Earth vector
+    - The z-axis is aligned with the component of the mean ecliptic pole at the observation time
+
+    Parameters
+    ----------
+    data: `~astropy.coordinates.BaseRepresentation` subclass instance
```

Comment:
All these parameters are shared between the frames, why don't you feed them into `add_common_docstring` to prevent from repeating yourself.

Reply:
Sure, I created a new issue (#3314) for this.

Reply:
To be clear, I want to implement this in a new PR because I want the approach to cover the other frames as well.

Reply:
Yeah I figured, seems wise.